testing/sync-test/src/main.rs

```diff
@@ -98,7 +99,13 @@ pub fn main() {
     let opts = Opts::from_args();
     println!("### Running sync integration tests ###");
     init_testing();
-    run_test_groups(&opts, vec![crate::logins::get_test_group()]);
-    run_test_groups(&opts, vec![crate::tabs::get_test_group()]);
-    println!("### Sync integration tests passed!");
+    // Commenting these out to make the test runs shorter and less
+    // noisy. It would be nice to add an option to `Opts` to only
```

Comment:
I've definitely wanted this too. Note: Don't land this code without uncommenting these!!